code/modules/holodeck/items.dm

```diff
@@ -7,64 +7,29 @@
 // Items
 //
 
-/obj/item/holo
-	damtype = STAMINA
-
-/obj/item/holo/esword
+/obj/item/melee/energy/sword/holographic
```

Comment:
This probably needs its bleedforce set to 0

Reply:
ded0212